test/linalg/cholesky.jl

```diff
@@ -94,18 +91,12 @@ using Base.LinAlg: BlasComplex, BlasFloat, BlasReal, QRPivoted, PosDefException
         U = Bidiagonal([2,sqrt(eltya(3))],[-1],true) / sqrt(eltya(2))
         @test full(chol(S)) ≈ full(U)
 
-        #lower Cholesky factor
```

Comment:
I thought it only applied to the tests which are now deprecated? Both `Hermitian(A, :U)` and `Hermitian(A, :L)` are tested in the tests just below.